### PR TITLE
merge: support ignore param to diff function

### PR DESCRIPTION
**Pull request by mafrosis**
My use-case requires the `ignore` param when diffing dictionaries.

I started using `Merger` today, and found this option was missing in that class.

### CODE REVIEW

**Review by joesolly — APPROVED** [review]

**Comment by mafrosis** [comment]
Unfortunately I trashed my branch after @joesolly left his approval 😄 

The code is fixed now - and it's quite a trivial change - looking forward to seeing this in a release sometime soon!

**Review by joesolly — APPROVED** [review]

**Review by mikaelho — COMMENTED** [review]
Nice and simple fix with a test.

While I am not a maintainer, I would think that also adding an update to the docs (updating the docstring) would make this PR easier to merge.

**Comment by mafrosis** [comment]
@jirikuncar What's stopping this from merging? Docs? Anything else?

File: dictdiffer/merge.py
Comment (jirikuncar) [inline dictdiffer/merge.py:40]: please add `ignore` to docstring

```
: param ignore: ....
```

**Review by jirikuncar — CHANGES_REQUESTED** [review]
Docstring needs to be updated. Otherwise it looks good 👍

**Comment by mafrosis** [comment]
Docstring added 🌞

**Comment by jayvdb** [comment]
ping @markgras @jirikuncar @adrien-berchet

**Review by jayvdb — APPROVED** [review]

**Review by adrien-berchet — APPROVED** [review]
Hi there,
I never used the `Merger` class but that looks like a missing feature indeed.
LGTM!

**Comment by markgras** [comment]
> ping @markgras @jirikuncar @adrien-berchet

Not too sure why I was included, but I will nevertheless give my two cents. Clean changes with no style guide infractions, updated docstring, and new unittest. Excellent contribution!

**Review by jirikuncar — APPROVED** [review]

**Pull request merged**
